apps/api/src/server/applications/application/representations/status/status.validator.js

```diff
@@ -0,0 +1,36 @@
+import { composeMiddleware } from '@pins/express';
+import { body } from 'express-validator';
+import { validationErrorHandler } from '../../../../middleware/error-handler.js';
+import { representationsStatusesList } from '../representation.validators.js';
+
+const invalidReasons = ['Duplicate', 'Merged', 'Not relevant', 'Resubmitted', 'Test'];
+const referredTo = ['Case Team', 'Inspector', 'Central Admin Team', 'Interest Party'];
```

Comment:
Its that Interested Party?

Reply:
yes... will change.

Reply:
@Philip-Philippou1 your call.  keep or change to snake case.

Reply:
these are more descriptive fields rather that status. I think they stay as is. 

Rep type is a good example of this (with symbols in certain places): 

```
const repTypes = [
 'Local authorities',
 'Members of the public/businesses',
 'Non-statutory organisations',
 'Statutory consultees',
 'Parish councils'
];
```